### PR TITLE
Cypress. Fix case "Image rotate" for Firefox.

### PR DESCRIPTION
**Pull request by dvkruchinin**
### Motivation and context
Patches are designed to solve the error 

```
<div class="ant-popover cvat-rotate-canvas-popover ant-popover-placement-right " style="left: 40px; top: 202px; transform-origin: -4px 50% 0px; pointer-events: none;">...</div>
          pointer-events: none` prevents user mouse interaction.
```
when running the test by Firefox with headless mode.
Link to check: https://github.com/dvkruchinin/cvat/actions/runs/595622158

### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2021 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

File: tests/cypress/integration/actions_tasks_objects/case_5_image_rotate.js [hunk 1]
```diff
@@ -31,7 +32,8 @@ context('Check if the image is rotated', () => {
             .then(($styles) => {
                 scaleAfter = Number($styles.match(/scale\((\d\.\d+)\)/m)[1]);
                 cy.expect(scaleBefore).to.be.greaterThan(scaleAfter);
-                cy.get('#cvat_canvas_content').dblclick();
+                cy.get('#cvat_canvas_content').dblclick().wait(500);
```
Comment (nmanovic) [inline tests/cypress/integration/actions_tasks_objects/case_5_image_rotate.js:35]: Why not 300 or 10000?

**Review by nmanovic — CHANGES_REQUESTED** [review]

File: tests/cypress/integration/actions_tasks_objects/case_5_image_rotate.js (hunk 1 above)
Comment (dvkruchinin) [reply tests/cypress/integration/actions_tasks_objects/case_5_image_rotate.js]: Reduced the waiting time. Also separated this behavior depending on the browser. This time was revealed when checking the passing of the test in CI when using the Firefox browser. At the moment, the Cypress repository has an open issue on the issue of different test results in the Firefox browser in headless mode and without it. https://github.com/cypress-io/cypress/issues/8610
If necessary, can I open a separate issue and specify the CVAT repository to reproduce the error?

**Comment by bsekachev** [comment]
@dvkruchinin 

What is wrong with this test? It works for me in current implementation (Firefox 85, headless).
Generally speaking, I am not sure we should rely on magic timeouts when testing UI.

**Comment by dvkruchinin** [comment]
> @dvkruchinin
> 
> What is wrong with this test? It works for me in current implementation (Firefox 85, headless, antd 4.12).
> Generally speaking, I am not sure we should rely on magic timeouts when testing UI.

This test does not always fail. But in CI-nightly, it is steadily falling. I will try to consider other solutions.

**Comment by dvkruchinin** [comment]
Revised the text. Removed wait() and added an extra click on the canvas to hide the popover before rotating the image.

**Comment by bsekachev** [comment]
@dvkruchinin 

Great, thanks!

**Review by bsekachev — APPROVED** [review]

**Pull request merged**